react/ui-components/src/hoc/BoundaryFilter.js

```
@@ -0,0 +1,741 @@
+              const highestIndex = hierarchy?.findIndex((item) => item?.boundaryType === props?.levelConfig?.highestLevel);
+              return highestIndex <= index && index <= lowestIndex;
+            })?.map((item) => {
+
+
+              return (item?.boundaryType === rootBoundaryType) ? (
+                <LabelFieldPair className="boundary-item" vertical={!updatedLayoutConfig?.isLabelFieldLayoutHorizontal}>
+                  <CardLabel className={"boundary-selection-label"}>
+                    {item?.boundaryType}
+                  </CardLabel>
+                  <div className="digit-field-full">
+                    {!(props?.levelConfig?.isSingleSelect && props?.levelConfig?.isSingleSelect?.includes(item?.boundaryType)) ?
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)